pyfr/writers/nodemaps.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+
+class VTKHONodeMaps(object):
```

Comment:
Would it be possible to go up to 8th order for the mappings (assuming it is supported by ParaView)?  For the mesh node maps we go up to 6, but it is not unheard of for solution orders to be slightly higher than mesh orders.

Reply:
VTK supports arbitrary orders for non-linear cells but Paraview limits the `Nonlinear subdivision level` parameter to 4. The true meaning of this parameter if a bit unclear to me, but it is supposed to recursively divide non-linear cells in triangles `Nonlinear subdivision level` times. Then the Lagrange basis of the considered cells are utilized to interpolate to those internal points of the subdivision triangles. Anyhow, I could add compatibility for such high-order (>6) elements if you wish.

Reply:
I think this is worthwhile; not because I expect it to be useful now but because I expect ParaView to improve its support for high-order elements in the future and so it is worth getting the support wired up now.

Reply:
Ok I'll regenerate the node maps. The current implementation is compatible with degree 5 hexahedrons. If I'm not mistaken, you want the implementation to be compatible with tensor-product elements up to degree 7 ?